react/components/ShippingOptionList.tsx

```diff
@@ -66,7 +76,12 @@ const ShippingOptionList: React.FC<Props> = ({
             <div className="flex w-100">
               <div className="flex flex-column w-100">
                 <span className="c-on-base">{deliveryOption.id}</span>
-                <span className="c-muted-1">
+                <span
+                  className={classnames({
+                    'c-muted-1': deliveryOption.id !== fastestOption?.id,
+                    'c-success': deliveryOption.id === fastestOption?.id,
+                  })}
```

Comment:
If multiple SLAs have the fastest delivery estimate, you will only highlight one of them, but I think you should highlight them all.

Reply:
ok!

Reply:
done in 
440c607